### DISCUSSION TITLE
Livewire In Porduction

### DISCUSSION THREAD

**Discussion by Nasim-Imtiaz** [livewire/livewire#4131] [1. Help]
Hi there, 

I am facing issues on production to server. 

In my local machine, the code is working smoothly and response time is quite good. But when I upload to amazon lightsail its response is more than 6 times.

The code is that 3 buttons and when one is clicked static data showed. nothing else. No database. 

What to do? Any suggestion?

**Comment by joshhanley** [comment]
@Nasim-Imtiaz hard to say, you will need to investigate you server logs to see what might be slowing it down, or work out whether its a network latency issue.

If you have any other servers setup, like in EC2, maybe you could also test it on there to see if it's a problem with lightsail specifically or something else.

Hope this helps!
